### PR TITLE
feat: kick command

### PR DESCRIPTION
**Pull request by NicolasLopes7**

### CODE REVIEW

File: cmd/client/main.go [hunk 1]
```diff
@@ -86,6 +87,22 @@ func main() {
 	}
 }
 
+func GetMessage(line string) *protocol.Message {
+	parts := strings.SplitN(line, " ", 2)
```
Comment (vit0rr) [inline cmd/client/main.go:91]: why not?

```suggestion
	parts := strings.Split(line, " ")
```

File: cmd/client/main.go [hunk 2]
```diff
@@ -86,6 +87,22 @@ func main() {
...
+	command := parts[0]
+
+	if strings.HasPrefix(command, "/") {
+		switch command {
+		case "/kick":
+			return protocol.NewMessage(protocol.KickUser, parts[1])
+		}
+	} else {
+		return protocol.NewMessage(protocol.SendMessage, line)
+	}
```
Comment (vitorsouzaalmeida) [inline cmd/client/main.go]: Why are you checking if have some prefix? The default case can be the `return protocol.NewMessage(protocol.SendMessage, line)`

```go
func GetMessage(line string) *protocol.Message {
	parts := strings.SplitN(line, " ", 2)
	command := parts[0]

	switch command {
	case "/kick":
		return protocol.NewMessage(protocol.KickUser, parts[1])
	default:
		return protocol.NewMessage(protocol.SendMessage, line)
	}
}
```

File: cmd/client/main.go [hunk 3]
```diff
@@ -86,6 +87,22 @@ func main() {
...
+
+	return nil
```
Comment (vitorsouzaalmeida) [inline cmd/client/main.go]: then you can remove it

File: cmd/server/main.go [hunk 4]
```diff
@@ -104,6 +104,16 @@ func (s *Server) Handle(conn net.Conn) {
...
+
+		case protocol.KickUser:
+			for addr, client := range s.clients.Clients {
+				if client.Name == m.Payload {
+					fmt.Printf("Kicking %s\n", addr)
+					(*client.Conn).Write(protocol.NewMessage(protocol.Die, "You were kicked").ToBytes())
+					(*client.Conn).Close()
+					s.clients.Delete(addr)
+				}
+			}
```
Comment (vitorsouzaalmeida) [inline cmd/server/main.go:122]: nice

**Pull request merged**